stonesoup/measures.py

```diff
@@ -8,6 +8,58 @@
 from .types.state import State
 
 
+class GenericMeasure(Base):
+    """Measure base type
+
+    A measure provides a means to assess the seperation between two
+     objects item1 and item2.
+    """
+
+    @abstractmethod
+    def __call__(self, item1, item2) -> float:
+        r"""
+        Compute the distance between a pair of objects
+
+        Parameters
+        ----------
+        item1 :
+        item2 :
+
+        Returns
+        -------
+        float
+            distance measure between a pair of input objects
+
+        """
+        return NotImplementedError
+
+
+class TrackMeasure(GenericMeasure):
```

Comment:
What is the need for TrackMeasure as well as GenericMeasure, they look the same to me?

Reply:
They are the same really. TrackMeasure is just an abstract class to it make clear which measure is needed for track association